File: APIQuanLyCuaHang/APIQuanLyCuaHang/ViewModels/DataChart/TopSellingProductsData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace APIQuanLyCuaHang.ViewModels.DataChart
{
    public class TopSellingProductsData
    {
        public List<ProductStatistics>? Products { get; set; }
    }

    public class ProductStatistics
    {
        public string? ProductId { get; set; }
        public string? ProductName { get; set; }
        public int Quantity { get; set; }
        public decimal TotalRevenue { get; set; }
    }
}

File: QuanLyCuaHangUXUI/QuanLyCuaHangProject/src/models/dto/dataCharts/TopSellingProductsData.js
export default class TopSellingProductsData {
  constructor(products = null) {
    this.products = products // products là một mảng các đối tượng ProductStatistics
  }
}

export class ProductStatistics {
  constructor(productId = null, productName = null, quantity = 0, totalRevenue = 0.0) {
    this.productId = productId
    this.productName = productName
    this.quantity = quantity
    this.totalRevenue = totalRevenue
  }
}
